src/GifWin/GifWitConversionProgressWindow.xaml.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Windows;
+using GifWin.ViewModels;
+
+namespace GifWin
+{
+    /// <summary>
+    /// Interaction logic for GifWitConversionProgressWindow.xaml
+    /// </summary>
+    public partial class GifWitConversionProgressWindow : Window
+    {
+        public GifWitConversionProgressWindow()
+        {
+            InitializeComponent();
+        }
+
+        protected override void OnContentRendered(EventArgs e)
+        {
+            ((GifWitConversionProgressViewModel)DataContext).ConvertLibrary();
```

Comment:
You (presumably) only want the task kicked off after the windows shows because you have tied the task to UI. If you decouple it (talked about in another comment), it won't be an issue.